### PR TITLE
Added web console steps to create subscription for logging

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill PTAL I added the steps to use the OpenShift web console to create the cluster logging subscription. What else is required to install cluster logging? What is no longer needed.

**Comment by mburke5678** [comment]
@jcantrill  PTAL I added the steps to use the OpenShift web console to create the cluster logging subscription. What else is required to install cluster logging? What is no longer needed.

File: modules/efk-logging-deploy-subscription.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// * logging/efk-logging-deploy.adoc
+
+[id='efk-logging-deploy-subscription_{context}']
+= Installing the cluster logging subscription
+
+You can use the {product-title} console to subcribe to a cluster logging subscription, which creates the Cluster Logging Operator.
+
+.Procedure
+
+To subscribe to the subscription:
```
Comment (jcantrill) [inline modules/efk-logging-deploy-subscription.adoc]: We'll need to revisit this once various changes to OLM merge.  The workflow is different.

**Comment by mburke5678** [comment]
@jcantrill Are all these tasks still required to install cluster logging?

https://logging-deploy-subscription--ocpdocs.netlify.com/openshift-enterprise/latest/logging/efk-logging-deploy.html

**Comment by jcantrill** [comment]
> @jcantrill Are all these tasks still required to install cluster logging?
> 
> https://logging-deploy-subscription--ocpdocs.netlify.com/openshift-enterprise/latest/logging/efk-logging-deploy.html

No.  The primary goal of operators is to make it easy.  We'll need to revisit this to clarify.  I suspect there will be a lot of "gutting" of the documentation.

**Comment by mburke5678** [comment]
Merging at the request of Jeff Cantrill to facilitate logging docs review.

**Pull request merged**